pre_commit/languages/rust.py

```diff
@@ -130,6 +133,12 @@ def install_environment(
     cli_deps = {
         dep for dep in additional_dependencies if dep.startswith('cli:')
     }
+    # Features starts with "--feature="
+    features = {
+        dep.replace(FEATURE_PREFIX, '')
```

Comment:
I don't think you should need a `replace(...)` -- we should be able to pass along arguments verbatim I would hope?

Reply:
What I've implemented uses a custom syntax, so e.g. `--feature=foo --feature=bar` is translated as `--features foo bar` when invoking Cargo.
If I understood you correctly you want to receive arbitrary Cargo arguments in `additional_dependencies` so we could pass, for example, `--features foo bar --locked`.

Is that what you're suggesting?

In that case, every value in `additional_dependencies` after a flag will be interpreted as a Cargo argument, because after a `--features` flag we cannot distinguish between a feature name or a dependency name.